app/utils/validators/forms/FieldValidator.scala

```diff
@@ -151,8 +151,12 @@ object FieldValidator {
   private val emailPattern = Pattern.compile("""^\S+@\S+$""")
   val isValidEmail: String => Boolean = (name: String) => emailPattern.matcher(name).matches()
 
-  private val eoriPattern = Pattern.compile("^[a-zA-Z]{2}[\\da-zA-Z]+$")
-  val isValidEORIPattern: String => Boolean = (name: String) => eoriPattern.matcher(name).matches()
+  private val eoriDigitsAmountMin = 10
```

Comment:
Now I'm wondering, because when I looked more I found that it can be 2 country characters plus 8 digits too.

https://en.wikipedia.org/wiki/VAT_identification_number